cookbooks/bcpc/templates/default/haproxy.cfg.erb

```diff
@@ -10,11 +10,13 @@ global
   group  haproxy
   pidfile  /var/run/haproxy.pid
   log /dev/log local0 info
+  # jmx page that is used for httpchk is more than the default size 16384 bytes
+  tune.chksize <%= @haproxy_tune_chksize %>
 
 defaults
   log  global
   maxconn  8000
-  source <%="#{node[:bcpc][:management][:vip]}"%>
+  source <%= @local_ip %>
```

Comment:
Yea! This allows us to properly health-check MySQL again?

Reply:
yes